components/navbar/Navbar.js

```diff
@@ -172,11 +172,10 @@ export default function Navbar() {
         </div>
 
         <div
-          className={`${
-            isOpen
+          className={`${isOpen
               ? "transform translate-y-0 opacity-100"
               : "transform -translate-y-96 opacity-0 "
-          } md:hidden z-20 absolute t-0 bg-gray-800 transition-all duration-700 ease-in-out  w-full`}
+            } md:hidden z-20 absolute t-0 bg-gray-800 transition-all duration-700 ease-in-out  w-full`}
```

Comment:
I think all the changes in this file are just formatting changes? To keep the PR clean please only include code changes - formatting changes can come in another PR

Reply:
I did created another pr for this, but I think I did some mistake here, the formatting changes are reflecting in both branches, I'll fix it.

Reply:
ok thanks 👍